docker-compose.yml

```diff
@@ -24,6 +24,20 @@ services:
       - static_volume:/app/experimenter/served/
     command: bash -c "/app/bin/wait-for-it.sh kinto:8888 -- python bin/setup_kinto.py;/app/bin/wait-for-it.sh db:5432 -- python /app/manage.py collectstatic --noinput;python /app/manage.py runserver 0:7001"
 
+  yarn-nimbus-ui:
+    image: app:build
+    env_file: .env
+    environment:
+      - CHOKIDAR_USEPOLLING=true
+    tty: true
+    ports:
+      - "3000:3000"
+    volumes:
+      - ./app:/app
+      - /app/node_modules/
+      - /app/experimenter/nimbus-ui/node_modules/
```

Comment:
No I think you wanna keep this cuz you wanna use the node modules that are built inside the container, not the outside ones